### PR TITLE
Attendance test case rename to snake_case

### PR DESCRIPTION
**Pull request by meisbokai**

### CODE REVIEW

**Comment by lucdx** [comment]
Yours also failing Checkstyle on Travis tho, and it's small letters for the first word

File: test/java/seedu/addressbook/logic/AttendanceTest.java [hunk 1]
```diff
@@ -101,14 +101,14 @@ public void setUp() throws Exception {
      */
 
     @Test
-    public void executeUpdateAttendanceInvalidArgsFormat() throws Exception {
+    public void execute_updateAttendance_invalidArgsFormat() throws Exception {
```
Comment (micklwl) [inline test/java/seedu/addressbook/logic/AttendanceTest.java]: The format for test is
featureUnderTest_testScenario_expectedBehavior().

Comment (meisbokai) [reply test/java/seedu/addressbook/logic/AttendanceTest.java]: so like updateAttendance_execute_invalidArgsFormat?

Comment (meisbokai) [reply test/java/seedu/addressbook/logic/AttendanceTest.java]: ok so is :
executeUpdateAttendance_invalidArgs_invalidArgsMessage

Comment (micklwl) [reply test/java/seedu/addressbook/logic/AttendanceTest.java]: yes

**Pull request merged**